WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -278,6 +278,20 @@ def update_events
     end
   end
 
+  def update_wcif
+    @competition = competition_from_params
+    wcif = params.permit!.to_h
```

Comment:
> Is the ambiguity here whether this is a string, or a ruby object?

Mostly that this data comes from `params`, which I assume means it could be anything from anyone. We _hope_ it turns out to be a valid WCIF document, but it could just as well be a description of Lady Gaga's tour schedule.

Also, calling something "wcif" is very vague. There are many possible WCIF documents in the world. Which one is this?